.circleci/workflows.yml

```diff
@@ -1997,13 +2061,58 @@ jobs:
     resource_class: << parameters.resource_class >>
     steps:
       - restore_cached_workspace
-      - build-binary
+      - check-if-binary-exists
+      - build-and-package-binary
       - build-cypress-npm-package:
           executor: << parameters.executor >>
-      - verify_should_persist_artifacts
       - upload-build-artifacts
+      - verify_should_persist_artifacts
```

Comment:
Seems like `verify_should_persist_artifacts` should still happen before `upload-build-artifacts`?

Reply:
yeah, this was a mistake

Reply:
Fixed this